### ISSUE TITLE
Pre-trained caffe model for FCN score evaluation

### ISSUE THREAD

**Issue by My-Consciousness** [phillipi/pix2pix#141]
I tried your evaluation code. I change the parameter --result_dir in ./scripts/eval_cityscapes/evaluate.py to the original val set of Cityscapes. I have resized all the original Cityscapes 
images(*_gtFine_labelIds.png and *_leftImg8bit.png) to 256x256, but the segmentation results become so bad as follows：
Mean pixel accuracy: 0.394909
Mean class accuracy: 0.057150
Mean class IoU: 0.025087
************ Per class numbers below ************
road           : acc = 0.999244, iou = 0.390848
sidewalk       : acc = 0.001727, iou = 0.001507
building       : acc = 0.062116, iou = 0.061576
wall           : acc = 0.000011, iou = 0.000011
fence          : acc = 0.000008, iou = 0.000008
pole           : acc = 0.000036, iou = 0.000036
traffic light  : acc = 0.000000, iou = 0.000000
traffic sign   : acc = 0.000000, iou = 0.000000
vegetation     : acc = 0.000003, iou = 0.000003
terrain        : acc = 0.000000, iou = 0.000000
sky            : acc = 0.022368, iou = 0.022336
person         : acc = 0.000000, iou = 0.000000
rider          : acc = 0.000000, iou = 0.000000
car            : acc = 0.000329, iou = 0.000322
truck          : acc = 0.000000, iou = 0.000000
bus            : acc = 0.000000, iou = 0.000000
train          : acc = 0.000000, iou = 0.000000
motorcycle     : acc = 0.000000, iou = 0.000000
bicycle        : acc = 0.000000, iou = 0.000000
Is there anything wrong with this fcn-8s-cityscapes.caffemodel ?

**Comment by junyanz** [comment]
Please see these two discussions [1](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/issues/150) and [2](https://github.com/phillipi/pix2pix/issues/116) 
@tinghuiz
